src/work-allocation/containers/task-home/task-home.component.ts

```diff
@@ -1,7 +1,9 @@
-import { Component, OnDestroy, OnInit } from '@angular/core';
+import { Component, OnChanges, OnDestroy, OnInit } from '@angular/core';
```

Comment:
`OnChanges` imported but not used?

Reply:
please remove it. Again agreed with Paul G